test/livebook/intellisense_test.exs

```diff
@@ -2024,7 +2024,13 @@ defmodule Livebook.IntellisenseTest do
       assert %{
                label: "hello/1",
                kind: :function,
-               documentation: "No documentation available",
+               documentation: """
+               No documentation available
+
+               ```
+               RemoteModule.hello(arg1)
```

Comment:
It could also be `RemoteModule.hello/1`, but I slightly prefer arg list for consistency.

Reply:
+1